data-es6.js

```diff
@@ -1059,6 +1082,28 @@ exports.tests = [
       },
     },
     {
+      name: 'for-loop iteration scope',
```

Comment:
I added the renaming. As for `false` entries it seems more involved as I don't see a single `false` entry related to a browser without a future `true` or `flag` entry. It seems a separate script-generated commit that would fill in all those missing entries would be best here. At least for web browsers it should be quite safe as their scores are checked by many people so someone would notice a mistake.